app/src/main/java/com/fivesysdev/standwithukraine/ui/StatisticFragment.kt

```diff
@@ -62,7 +61,7 @@ class StatisticFragment : Fragment(R.layout.fragment_statistic) {
     private fun setDayStatistic(dayStatistic: DayStatistic?) {
         checkBlockNextButton()
         if (dayStatistic != null) {
-            binding.recyclerview.adapter = StatisticAdapter(dayStatistic.statistic)
+            binding.recyclerview.adapter = StatisticAdapter(dayStatistic.getStatisticsPair())
```

Comment:
Why do you create a new adapter each time the statistics change?

Reply:
I need somehow change the list of elements. So I was thinking about creating a new adapter each time or adding a new method to the adapter class, that replaces data in the list. The first option seemed the simplest, but not the smartest I guess.

Reply:
there are a bunch of methods like notifyDataSetChanged for this purpose.
Please, check the documentation/articles on how to properly change data in list

Reply:
Okay, I'll check